test-suite/src/schemaless/basic.rs

```diff
@@ -0,0 +1,115 @@
+use {
+    crate::*,
+    gluesql_core::prelude::Value::{self, *},
+    serde_json::json,
+};
+
+test_case!(basic, async move {
+    run!("CREATE TABLE Player");
+    run!(format!(
+        "INSERT INTO Player VALUES ('{}'), ('{}');",
+        json!({ "id": 1001, "name": "Beam", "flag": 1 }),
+        json!({ "id": 1002, "name": "Seo" }),
+    )
+    .as_str());
+
+    run!("CREATE TABLE Item");
+    run!(format!(
+        "INSERT INTO Item VALUES ('{}'), ('{}');",
+        json!({
+            "id": 100,
+            "name": "Test 001",
+            "dex": 324,
+            "rare": false,
+            "obj": {
+                "cost": 3000
+            }
+        }),
+        json!({
+            "id": 200
+        })
+    )
+    .as_str());
+
+    test!(
+        "SELECT name, dex, rare FROM Item WHERE id = 100",
```

Comment:
Is there a way to filter with Map like this?
```sql
gluesql> SELECT * FROM Item WHERE obj = '{"cost": 3000}';
-- currently 0 rows
```

Reply:
PartialEq between `Value::Map(_)`s is implemented so it can be supported. we can consider using `CAST` for now.
```sql
SELECT * FROM Item WHERE obj = CAST('{ "cost": 3000 }' AS MAP);
```
Not supported yet, we need to add `MAP` cast to `Value::try_cast_from_literal`.